blockly-ws-search/webpack.config.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+/**
+ * @fileoverview Webpack configuration file.
+ * @author aschmiedt@google.com (Abby Schmiedt)
+ */
+
+const path = require('path');
+const webpack = require('webpack');
+
+module.exports = env => {
+    return {
+        target: 'web',
+        mode: env.mode,
+        entry: './src/WorkspaceSearch.js',
+        devtool: 'source-map',
+        output: {
+            path: path.resolve(__dirname, 'dist'),
+            filename: 'workspace-search.umd.js',
+            libraryTarget: 'umd',
+            umdNamedDefine: true,
```

Comment:
What does umdNamedDefine do?

Reply:
Originally, it was used to fix one of our examples. But it seems to be working without it so will remove.